src/external/etsf_nanoquanta.jl

```diff
@@ -164,14 +164,14 @@ function load_density(T, folder::EtsfFolder; magnetic_moments=[])
         ρtot_real = Array{T}(folder.den["density"][1, :, :, :, 1])
         @assert basis.fft_size == size(ρ_real)
 
-        return from_real(basis, ρtot_real), nothing
+        return ρ_from_total_and_spin(ρtot_real)
```

Comment:
This is a bug: Second argument missing.

Reply:
Indeed, well spotted. Should be fixed by switching to 
```
function ρ_from_total_and_spin(ρtot, ρspin=nothing)
```